tracer/src/Datadog.Tracer.Native/tracer_tokens.cpp

```diff
@@ -351,7 +421,26 @@ HRESULT TracerTokens::WriteBeginMethod(void* rewriterWrapperPtr, mdTypeRef integ
     {
         const auto [elementType, argTypeFlags] = methodArguments[i].GetElementTypeAndFlags();
 
-        if (enable_by_ref_instrumentation && (argTypeFlags & TypeFlagByRef))
+        bool isByRefLike = false;
```

Comment:
I see that we handle ref struct parameter in the fast path, what about slow path? since ref struct can't be boxed, do we need to do something about it in the slow path?

Reply:
True this PR adds support for the fast path only, I'll add support for the slow path later in another PR.

Reply:
I understand. Should we at least disable instrumentation for the slow path for now in the ref struct scenario or is it fine as it is now? Anyway, you can resolve it, I just wanted to point it out.